app/Http/Controllers/API/v1/Category/CategoryController.php

```diff
@@ -149,4 +149,11 @@ public function listOfCategories(Request $request)
 
         return $this->paginate($categories->get());
     }
+
+    public function getUnpaginatedCategoryList() 
+    {
+        $categories = Category::all();
+
+        return $this->showAll($categories);
+    }
```

Comment:
instead of creating another function, can we use the existing one and just have a condition if we will return paginated/unpaginated?